### ISSUE TITLE
Error when running npm start

### ISSUE THREAD

**Issue by whoabuddy** [Daemon-Technologies/Mining-Local-Server#13]
Experienced this while walking a user through the installation (Mac OSX 10.15.7), and was able to replicate on my machine (Linux Mint 19).

Steps to reproduce:
1. Download Mining-Local-Server via git
2. cd into Mining-Local-Server folder
3. npm install
4. npm start

Error:
```
> miningbot-server@1.0.0 start /home/absorb/sync/Consulting/Daemon Technologies/Mining-Local-Server
> node server.js

/home/absorb/sync/Consulting/Daemon Technologies/Mining-Local-Server/server.js:1
import express from "express"
^^^^^^

SyntaxError: Cannot use import statement outside a module
`

**Comment by gavinisagi** [comment]
No error execute commands above. You can check the version of node & npm.
Here is my version:
```
$ node -v
v14.8.0
$ npm -v
6.14.7
```

**Comment by gavinisagi** [comment]
Closed by 1.0.0 release
